src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -71,45 +120,59 @@ def create_container(client,
                                                               username=registry_username,
                                                               password=registry_password)]
     elif ACR_SERVER_SUFFIX in image:
-        if registry_password is None:
+        if not registry_password:
             try:
                 registry_password = prompt_pass(msg='Image registry password: ')
             except NoTTYException:
                 raise CLIError('Please specify --registry-password in non-interactive mode.')
 
         acr_server = image.split("/")[0] if image.split("/") else None
         acr_username = image.split(ACR_SERVER_SUFFIX)[0] if image.split(ACR_SERVER_SUFFIX) else None
-        if acr_server is not None and acr_username is not None:
+        if acr_server and acr_username:
             image_registry_credentials = [ImageRegistryCredential(server=acr_server,
                                                                   username=acr_username,
                                                                   password=registry_password)]
         else:
             raise CLIError('Failed to parse ACR server or username from image name; please explicitly specify --registry-server and --registry-username.')
 
-    command = None
-    if command_line is not None:
-        command = shlex.split(command_line)
+    return image_registry_credentials
 
-    container = Container(name=name,
-                          image=image,
-                          resources=container_resource_requirements,
-                          command=command,
-                          ports=[ContainerPort(port=port)],
-                          environment_variables=environment_variables)
 
-    cgroup_ip_address = None
-    if ip_address is not None and ip_address.lower() == 'public':
-        cgroup_ip_address = IpAddress(ports=[Port(protocol=ContainerGroupNetworkProtocol.tcp, port=port)])
+def create_azure_file_volume(azure_file_volume_share_name, azure_file_volume_account_name, azure_file_volume_account_key):
+    """Create Azure File volume. """
+    azure_file_volume = None
+    if azure_file_volume_share_name:
+        if not azure_file_volume_account_name:
+            try:
+                azure_file_volume_account_name = prompt(msg='Azure File storage account name: ')
+            except NoTTYException:
+                raise CLIError('Please specify --azure-file-volume-account-name in non-interactive mode.')
+        if not azure_file_volume_account_key:
+            try:
+                azure_file_volume_account_key = prompt_pass(msg='Azure File storage account key: ')
+            except NoTTYException:
+                raise CLIError('Please specify --azure-file-volume-account-key in non-interactive mode.')
```

Comment:
You should just fail if the parameters aren't set.
Unlike xplat CLI (the prev. Azure CLI), our pattern is not to prompt in this way. There are some exceptions e.g. for Password but really this sort of validation should be done in a validator or if not, just fail.